src/components/ContentNode.vue

```diff
@@ -468,6 +468,9 @@ function renderNode(createElement, references) {
           titleStyle: reference.titleStyle,
           hasInlineFormatting: !!titleInlineContent,
         },
+        // Inline links need to be underline under WCAG 2.2 compliance
+        // More info: https://www.w3.org/WAI/WCAG22/Techniques/failures/F73
+        class: 'underline',
```

Comment:
Could we just add a `:deep(a)` rule for any anchors within a `ContentNode` since we always want to apply the underline for links here? That would also catch the missed `InlineType.link` case from above which I believe you would also want to apply an underline.

Reply:
https://github.com/apple/swift-docc-render/pull/733/commits/8f20497b471af7444639cadc23888eaaa8f93665